locale/eu_ES.po

```diff
@@ -389,10 +390,10 @@ msgstr ""
 msgid "bonus %(words)d words"
 msgstr "bonus %(words)d hitz"
 
-#: ukbot/rules/templateremoval.py:88
+#: ukbot/rules/templateremoval.py:95
 #, python-format
-msgid "removal of {{tl|%(template)s}}"
-msgstr "{{tx|%(template)s}} txantiloia kentzea"
+msgid "removal of {{tl|%(template)s%(lang)s}}"
```

Comment:
This won't be correct without a `|`. Also, unfortunately, the `{{tl}}` template works differently in different languages – the feature of linking to other languages seems to be Norwegian-specific. I think it's better to use `[[Template:%(template)s]]`.

(This goes for all uses of `{{tl}}`, of course, not just this specific one.)